src/app/content/components/Footer.tsx

```diff
@@ -0,0 +1,409 @@
+import React, { Component } from 'react';
```

Comment:
cc is complaining about the length of this file, i've been doing a thing where i move the styled components to a different file in the format of 

`Footer/index.js` <- no styled components, just assemble the styled components and put any required js shenanigans
`Footer/styled.js` <- export the styled components, try not to put anything in here that isn't styled component definitions